src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -115,16 +115,14 @@ internal static NamespaceOrTypeSymbol ContainingNamespaceOrType(this Symbol symb
             return null;
         }
 
-        internal static Symbol ContainingNonLambdaMember(this Symbol containingMember)
+        internal static Symbol ContainingNonLambdaMember(this Symbol member)
```

Comment:
>internal static Symbol ContainingNonLambdaMember(this Symbol member) [](start = 8, length = 68)

Is there an intent to change behavior of this function? Please revert otherwise.